src/thunderbots/software/util/parameter/configuration.cpp

```diff
@@ -0,0 +1,63 @@
+#include "util/parameter/configuration.h"
+
+namespace Configuration
+{
+void updateParametersFromParameterServer()
```

Comment:
nit, but would rename to something like `updateParametersFromRosParameterServer` so it's clear what parameter server we're getting these from

Reply:
done